src/angular-app/languageforge/lexicon/core/lexicon-project.service.ts

```diff
@@ -100,10 +100,6 @@ export class LexiconProjectService {
     return this.api.call('lex_project_update', [settings], callback);
   }
 
-  updateSettings(smsSettings: any, emailSettings: any, callback?: JsonRpcCallback) {
-    return this.api.call('project_updateSettings', [smsSettings, emailSettings], callback);
-  }
-
   readSettings(callback?: JsonRpcCallback) {
```

Comment:
delete this method and the endpoint associated with it.

Reply:
done in [c34a6b6](https://github.com/sillsdev/web-languageforge/pull/1611/commits/c34a6b62da664b8d2bc08e7a13096984be0b2ea1)